src/cart/utils/memcheck-cart.supp

```diff
@@ -95,100 +35,3 @@
    ...
    fun:sock_pe_progress_thread
 }
-# We couldn't find a rte_log_fini() function or similar on SPDK codebase to
-# suppress the leak warning message. The rte_log_init() function is called
-# from a constructor while loading the SPDK libraries.
-{
-   spdk leak 0
-   Memcheck:Leak
-   match-leak-kinds: reachable
-   fun:calloc
-   fun:rte_log_init
-   ...
-}
```

Comment:
This worries me a bit, we put this in for a reason yet for some reason it's not failing when I removed it.  Do we still have the same coverage we used to?